flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableSinkITCase.java

```diff
@@ -322,6 +325,9 @@ void testBatchSink() throws Exception {
 
     @Test
     void testReadingFromChangelogSource() throws Exception {
+        if (!getMetadata().supportUpsert()) {
```

Comment:
Here the problem is the deletion no? (I have this same problem in Trino dialect)

Reply:
If I remove this judgment, this test case will fail. I would like to hear your opinion on how to handle this issue

Reply:
the approach is fine, I only dont know if the name of method is the correct (supportUpsert)..

Reply:
Ok, I will modify this issue later

Reply:
I have updated the method name